### PR TITLE
Update AutoHotkey language support

### PR DESCRIPTION
**Pull request by devPnal**
Hello.

This plugin is served on my repo. But, the Autohotkey v1 is departed now, So this plugin will not update in the future.
So, I moved the plugin to original repo(lite-xl/lite-xl-plugins) from my repo.

I deleted an asterisk(*) mark in README file, and uploaded the plugin file.

The file is totally same that before marged.

Please check it, sir.

### CODE REVIEW

**Comment by Guldoman** [comment]
Currently the `README` is autogenerated from `manifest.json`. Could you update that file instead?

File: plugins/language_autohotkey_v1.lua [hunk 1]
```diff
@@ -0,0 +1,155 @@
+-- mod-version:2 -- lite-xl 2.0
```
Comment (Guldoman) [inline plugins/language_autohotkey_v1.lua:1]: Could you update this with the new mod-version?
```suggestion
-- mod-version:3
```

**Comment by devPnal** [comment]
Sure. I'll commit soon. wait for a day please, sir.

**Comment by devPnal** [comment]
@Guldoman 
I updated manifest.json file. Do I need revert readme file? I think that's okay to leave it if it generated by menifest.json.

**Comment by Guldoman** [comment]
Looks good, thank you!

**Pull request merged**